scripts/bin_to_c.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
```

Comment:
I think we should start enforcing Python3 (because of [this](https://pythonclock.org/)), here it can be both Python2 or Python3.

Reply:
sure, my fault, i'll move to `/usr/bin/env python3`.